### DISCUSSION TITLE
Slashes in URL are encoded on Android app

### DISCUSSION THREAD

**Discussion by tez3998** [community/community#70547] [Mobile] [answered]
### Select Topic Area

Bug

### Body

I don't know if this is a bug or a specification, but I will report it.

When I see a file on GitHub app and copy that link, slashes in URL are not encoded as follows:
```
https://github.com/mdn/content/blob/main/files/en-us/web/html/element/img/clock-demo-200px.png
```

On the other hand, when I see a folder on GitHub app and copy that link, slashes in URL are encoded as follows:
```
https://github.com/mdn/content/tree/main/files%2Fen-us%2Fweb%2Fhtml%2Felement%2Fimg
```

Other information is as follows:

- OS: Android
- Version of Android: 13
- Version of GitHub app: 1.134.2

**Comment by elisealix22** [comment, marked as answer]
Hi @tez3998 , nice catch! Thank you for reporting this. I created an issue internally to track this discrepancy. Our Android team will take a look!

**Reply by tez3998** [reply]
Hi @elisealix22 , thank you for your response!

May I press the "Mark as answer" button?
Or should I not press the button so that we can communicate in the future?

**Reply by elisealix22** [reply]
I would say mark as answer! It's in our pipeline and we'll take a look soon!

**Reply by stevepopovich** [reply]
Hi @tez3998 at this time we are not going to take any action here. I can see that it encodes the URL differently, but both the App and the website handle these URLs correctly. If you find something broken here, please let us now!

**Reply by tez3998** [reply]
Hi @stevepopovich 
Thank you for taking the time to look into this. I understand your decision.
Have a great weekend!

**Reply by MulverineX** [reply]
@elisealix22 this was never actually fixed and has been in the app since its release years ago, is this actually on the team's radar? Its a huge pain especially with Java projects where there's lots of file path nesting nonsense
